pkgs/development/ruby-modules/gem/default.nix

```diff
@@ -142,6 +142,10 @@ stdenv.mkDerivation ((builtins.removeAttrs attrs ["source"]) // {
       gempkg=$(echo "$output" | grep -oP 'File: \K(.*)')
 
       echo "gem package built: $gempkg"
+    elif [[ "$type" == "git" ]]; then
+      git init
+      git add .
+      rm -rf .git/logs/ .git/hooks/ .git/index .git/FETCH_HEAD .git/ORIG_HEAD .git/refs/remotes/origin/HEAD .git/config
```

Comment:
Oh, really, you can remove `.git/index`? What's left in `.git` after this?

Reply:
:) I copied your reference above! I experimented directly, and removing the index clobbers the effect of `git add` (which is obvious in hindsight). Reversing the order fixes it.